var/spack/repos/builtin/packages/amdfftw/package.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+from spack.pkg.builtin.fftw import FftwBase
+
+
+class Amdfftw(FftwBase):
+    """FFTW (AMD Optimized version) is a comprehensive collection of
+    fast C routines for computing the Discrete Fourier Transform (DFT)
+    and various special cases thereof.
+
+    It is an open-source implementation of the Fast Fourier transform
+    algorithm. It can compute transforms of real and complex-values
+    arrays of arbitrary size and dimension.
+    AMD Optimized FFTW is the optimized FFTW implementation targeted
+    for AMD CPUs.
+
+    For \'single\' precision build, please use precision value as float.
+    Example : \'spack install amdfftw precision=float\'
```

Comment:
There's no need to escape the single quotes here.

Reply:
Remove escape sequence slash